src/platform/web/ui/session/room/PeekableRoomView.js

```diff
@@ -0,0 +1,41 @@
+import {TemplateView} from "../../general/TemplateView";
+import {TimelineView} from "./TimelineView";
+import {TimelineLoadingView} from "./TimelineLoadingView";
+import {AvatarView} from "../../AvatarView";
+
+export class PeekableRoomView extends TemplateView {
```

Comment:
It feels a bit weird to me that we now have both the `UnknownRoomView` and the `PeekableRoomView` but a single `UnknownRoomViewModel`.

To me what we're achieving in this PR is to add functionality to the screen that the user sees when trying to access a room they're not a member of. This new functionality makes it so that that screen shows the timeline of the room, when said room is peekable. Since it's the same screen, I think it would make sense to have all the logic and view-logic in the same place.

In short, I think it could make sense to include the view-logic in this file into `UnknownRoomView`, so that there is a single view and a single view-model for that screen. Sometimes it makes sense to split view-logic into separate components but, in this case, the resulting unified view would still be very simple so I'm not sure separating it brings advantages.